packages/hardhat/package.json

```diff
@@ -11,7 +11,8 @@
     "lint": "eslint --config ./.eslintrc.json --ignore-path ./.eslintignore ./*.ts ./deploy/**/*.ts ./scripts/**/*.ts ./test/**/*.ts",
     "lint-staged": "eslint --config ./.eslintrc.json --ignore-path ./.eslintignore",
     "test": "REPORT_GAS=true hardhat test --network hardhat",
-    "verify": "hardhat etherscan-verify"
+    "verify": "hardhat etherscan-verify",
+    "wagmi" : "hardhat run scripts/generateWagmiConfig.ts && wagmi"
```

Comment:
maybe create `wagmi-generate` script?

Reply:
You can also add `prettier --write wagmi.config.ts` after running script, so the config will look better

Reply:
Tysm @rin-st, always love your suggestions 🙌

updated at [ae4b077](https://github.com/scaffold-eth/se-2/pull/200/commits/ae4b077b03f74d702365c3003f81642fec4e4072)